client/src/components/Learning/Content/Content.js

```diff
@@ -2,7 +2,7 @@ import React, { useEffect } from "react";
 import "./Content.css";
 
 export default function Content(props) {
-  const { data, title, setChapterIndex } = props;
+  const { data, title, setChapterIndex, chapterId } = props;
   useEffect(() => {
     const headings = document.querySelectorAll(".learning-content h1");
```

Comment:
Adding a blog for you to read here, https://www.upbeatcode.com/react/how-to-use-queryselector-in-react/

Reply:
Read it!